### PR TITLE
Responsive design changes

### PR DESCRIPTION
**Pull request by ErikBerta**
Design changes, responsive design

screen shot 2015-08-22 at 14 14 39
screen shot 2015-08-22 at 14 14 24
screen shot 2015-08-22 at 14 13 49

screen shot 2015-08-22 at 14 13 26

### CODE REVIEW

File: app/templates/drivers/show.hbs [hunk 1]
```diff
@@ -56,13 +62,13 @@
...
-          <a {{bind-attr href=model.facebookUrl}}>{{fa-icon "facebook-square" size=1}}Profile</a><br>
+          <a {{bind-attr href=model.facebookUrl}}>{{fa-icon "facebook-square" size=1}}FB Profile</a><br>
         {{/if}}
         {{#if model.email}}
-          <a {{bind-attr href=model.mailToEmail}}>{{fa-icon "envelope-o" size=1}}{{model.mailToEmail}}</a>
+          <a {{bind-attr href=model.mailToEmail}}>{{fa-icon "envelope-o" size=1}}{{model.email}}</a>
         {{/if}}<br>
         {{#if model.phone}}
-          <a {{bind-attr href=model.callToPhone}}>{{fa-icon "phone" size=1}}{{model.callToPhone}}</a>
+          <a {{bind-attr href=model.callToPhone}}>{{fa-icon "phone" size=1}}{{model.phone}}</a>
```
Comment (flyfy1) [inline app/templates/drivers/show.hbs]: What's the difference between: `callToPhone` and `phone`?

Comment (ErikBerta) [inline app/templates/drivers/show.hbs]: phone generates just the normal phone number - 123 236 367... and callToPhone generates - tel:123 236 367

callToPhone: Ember.computed("phone", function() {
    return "tel:" + this.get("phone");
  })

Comment (flyfy1) [inline app/templates/drivers/show.hbs]: OK

**Comment by flyfy1** [comment]
:+1:

**Comment by yanghl22** [comment]
OK

**Pull request merged**